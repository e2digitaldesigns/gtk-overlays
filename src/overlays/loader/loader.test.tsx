import * as React from "react";
import { render } from "@testing-library/react";
import Loader from "./loader";

const mockedProps = {
  bgColor: "black",
  color: "white"
};
const setup = () => {
  return render(<Loader />);
};

afterEach(() => {
  jest.clearAllMocks();
});

describe("Loader", () => {
  it("should match snapshots", () => {
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });
});
