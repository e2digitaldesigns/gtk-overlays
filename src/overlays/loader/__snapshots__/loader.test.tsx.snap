// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match snapshots 1`] = `
<div>
  <div
    class="sc-bdvvtL dfAnzF"
  >
    <div
      class="sc-gsDKAQ edMtDB"
    >
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
